src/data/navigation/navigation.json

```diff
@@ -297,7 +272,7 @@
     }
   },
   "utilities": {
```

Comment:
```suggestion
  "patterns": {
```

Sorry I missed this the first time, we need to upate this also to get the links to update correctly.

Reply:
Fixed

Reply:
updated